sdk/Lusid.Sdk.Tests/Utilities/TestDataUtilities.cs

```diff
@@ -17,14 +14,14 @@ public static class TestDataUtilities
         public const string MarketDataScope = "FinbourneMarketData";
         public static string ValuationDateKey = "Analytic/default/ValuationDate";
         public static string InstrumentTag = "Analytic/default/InstrumentTag";
-        public static string HoldingPvKey = "Holding/default/PV";
+        public static string ValuationPvKey = "Valuation/PV/Amount";
         public static string InstrumentName = "Instrument/default/Name";
         
-        public static readonly List<AggregateSpec> valuationSpec = new List<AggregateSpec>
+        public static readonly List<AggregateSpec> ValuationSpec = new List<AggregateSpec>
```

Comment:
Consider putting a comment here describing what this is. I'm not sure, so others might not be either